sirepo/template/srw.py

```diff
@@ -1893,16 +1898,29 @@ def _rotate_report(report, ar2d, x_range, y_range, info):
     return ar2d, x_range, y_range
 
 
+def _rsopt_res_file_infix(elements, params):
+    s = ''
+    for e in elements:
+        for p in [p for p in params if p in e]:
+            for i, x in enumerate(e[p].fieldNames):
+                if e[p].offsets[i] != 0:
+                    s += f'{e.title}_{p}_{x}_'
```

Comment:
This is strange. Is there only one? The output will be a long string separated by underscores. No separator?

Reply:
Yes. This is the format used by the initial rsopt example (so e.g. ```beam_intensities_Aperture_size_horizontalSize_Aperture_size_verticalSize_10_runs.npy```). I suppose the idea is to have info about the run apparent at a glance but I can see it running into filename length limits in even moderately complex cases. May be worth getting input from @cchall...

Reply:
I'm not sure I know what the file being named here is supposed to be. Is this the libensemble history file generated at the end of an rsopt run?

Reply:
It's the SRW output, so it's nothing directly to do with rsopt. I think it came from an example of parallel execution - or maybe I invented it and forgot. Anyway it seems there is no reason for this format so I will just replace it with a static filename

Reply:
(it came from the ```SRW_multiprocess.py``` example in ```rslight```):

```
    ####################### updates where input/output will be saved based on which offsets 
    save_str = ''
    if offsets_mirror1:
        
        save_str += 'mirror1_offsets_'
        
    if offsets_mirror2:
        
        save_str += 'mirror2_offsets_'
...
    ###### actual input and output files used for ML 
    np.save('datasets/beam_intensities_' + save_str + str(n_runs) + 'runs.npy', beams_all)
```

Reply:
d4bb4d1